src/styles/annotator/highlights.scss

```diff
@@ -0,0 +1,24 @@
+@use '../variables' as var;
+
+.hypothesis-highlights-always-on {
```

Comment:
The `hypothesis-highlights-always-on` class is a bit special. It is added to / removed from the root element of the document when you toggle the visibility of highlights.

The general change here is from `annotator-$component` to `hypothesis-$component`. The prefix helps to avoid conflicts with the page's CSS and is helpful when you're looking at elements in the dev tools to know which things come from Hypothesis.

This comment reminded me that I need to surface this convention change more widely. I'll post a note in #eng-frontend.